pkg/openapi/crdSync.go

```diff
@@ -157,15 +158,29 @@ func (o *Controller) ParseCRD(crd unstructured.Unstructured) {
 
 	parsedSchema, err := openapi_v2.NewSchema(schema, compiler.NewContext("schema", nil))
 	if err != nil {
-		log.Log.Error(err, "could not parse crd schema", "name", crdName)
-		return
+		isOpenV3Error(err, crdName)
```

Comment:
What if it's not open v3 error? Would be good to have return value as boolean `func isOpenV3Error()bool{}` and handle it accordingly.

Reply:
Okay. I have made the changes accordingly.